SearchExampleSwift/APIKEY

```diff
@@ -0,0 +1 @@
+kla5ReI1XnDzOtqJoYcIxtSseCEgNtaf
```

Comment:
Committed key. Remove this and squash the commit where it was added so it's gone from history...